### ISSUE TITLE
Enabling MPI 

### ISSUE THREAD

**Issue by manodeep** [arhstevens/DarkSage#3]
DarkSage currently doesn't have MPI properly enabled and does not run on multiple nodes on ozstar. Porting over the sage style of MPI parallelisation should be fairly straightforward. 

@arhstevens Is there any reason that the sage-style MPI feature was removed from DarkSage?

**Comment by arhstevens** [comment]
I don't think it ever was "removed" (can't see anything obvious that would suggest it was after browsing the commit history).  I think it was simply a case of Dark Sage branching off SAGE before MPI was properly implemented for SAGE.  Getting this implemented properly has been on my to-do list.  Hasn't been a priority because it hasn't been a problem thus far.  Sounds like it is time to get it done!

**Comment by manodeep** [comment]
@arhstevens Looking at the source, the MPI seems to have been implemented properly. Is that not the case?

**Comment by arhstevens** [comment]
Give it a go and find out, I guess.  I never got it to work with the actual MPI flag on.  Has been quite a while since I tried though.

**Comment by manodeep** [comment]
From my read, the MPI is fully functional. You can also remove the calls to `time` and `sleep` from `main.c`. Should work just fine in MPI. 

If the MPI mode still doesn't work, please let me know. I am more than happy to fix it.
